app/v1/connectors/PaymentsConnector.scala

```diff
@@ -34,12 +36,14 @@ class PaymentsConnector @Inject()(val http: HttpClient,
 
   def retrievePayments(request: PaymentsRequest)(implicit hc: HeaderCarrier,
                                                  ec: ExecutionContext,
-                                                 userRequest: UserRequest[_]): Future[DesOutcome[PaymentsResponse]] = {
+                                                 userRequest: UserRequest[_],
+                                                 correlationId: String): Future[DesOutcome[PaymentsResponse]] = {
 
     import v1.connectors.httpparsers.StandardDesHttpParser._
-    implicit val requestToDate: String = request.to
+    implicit val requestToDate: LocalDate = LocalDate.parse(request.to)
```

Comment:
instead of doing the parsing here, could it be done in the request parser after validation?

Reply:
this is used in the response